parrot/src/pfs_service_local.cc

```diff
@@ -326,7 +303,7 @@ process to sleep and wait for actual input to become ready.
 
 	virtual int is_seekable()
 	{
-		return !is_a_pipe;
```

Comment:
This flag existed so that one gets a proper errno when attempting to seek on a pipe.  Does that issue no longer come up?